x-pack/plugins/uptime/server/lib/helper/get_filtered_query.ts

```diff
@@ -9,11 +9,15 @@ import { get, set } from 'lodash';
 export const getFilteredQuery = (
   dateRangeStart: string,
   dateRangeEnd: string,
-  filters?: string | null
+  filters?: string | null | any
```

Comment:
Hm - I don't know that we actually need `any` defined there. Good find. I will look into it.

Reply:
We do need `any` (or a more specific type at least, because the function handles multiple object types) for now, but like you say, we will eventually be able to refactor and delete this code. When we implement https://github.com/elastic/kibana/issues/29745 we won't need `any`.. anymore. When we update the GQL schema, we can use the type we generate from that.